gem/templates/profiles/viewprofile.html

```diff
@@ -14,27 +14,23 @@
         <hr>
         <div class="nickname-form">
             <div class="block">
-                <form method="post" id="frmProfileEdit"
-                      action="{% url 'molo.profiles:edit_my_profile' %}">
-                    {% csrf_token %}
-                    <h5>{% trans "Change your name" %}</h5>
-                    <p>{% trans "When you type comments your name will appear as:" %}
-                    <span>{{ user.profile.alias }}</span></p>
 
-                    <p>{% trans "Please do not use your real name, phone number or email address as it's unsafe." %}</p>
+                    <h3>{% trans "Your Profile" %}</h3>
 
-                    <p>{% trans "Type new display name:" %}</p>
-                    {{ edit_profile_form.alias }}
 
+                    <p>{% trans "When you type comments your name will appear as:" %}
```

Comment:
Can we change this copy if they haven't set a nickname?
 `You haven't set a nickname, so your comments will appear as "anonymous"`